package.json

```diff
@@ -26,6 +26,7 @@
     "acorn-numeric-separator": "^0.3.0",
     "acorn-static-class-features": "^0.2.1",
     "bindings": "^1.4.0",
+    "es-get-iterator": "^1.1.0",
```

Comment:
Should be `devDependencies` 🙂

Reply:
We're not currently pinning _any_ versions in this repo! I forget that's the preferred pattern, let's get a separate PR up for that.